### PR TITLE
fix(bingx) - nested orders parsing & JSON handling

### PR DESCRIPTION
**Pull request by ttodua**
fixes the parsing of response from createOrder, when order was submitted with attached SL/TP

### CODE REVIEW

File: ts/src/bingx.ts [hunk 1]
```diff
@@ -1903,13 +1903,28 @@ export default class bingx extends Exchange {
         //     }
         //
         if (typeof response === 'string') {
+            // broken api engine : order-ids are too long numbers (i.e. 1742930526912864656)
+            // and JSON.parse can not handle them in JS, so we have to use .parseJson
+            // however, when order has an attached SL/TP, their value types need extra parsing
+            response = this.fixStringifiedJsonMembers (response);
             response = this.parseJson (response);
```
Comment (carlosmiei) [inline ts/src/bingx.ts:1910]: @ttodua I don't think this is working
image

Comment (ttodua) [reply ts/src/bingx.ts:1910]: oh. can you tell  which command did you run and what response did you get initially?

Comment (carlosmiei) [reply ts/src/bingx.ts:1910]: @ttodua I ran something like this:
`n bingx createOrder "LTC/USDT:USDT" market buy 1 undefined '{"takeProfit":{"stopPrice":100},"stopLoss":{"stopPrice": 50}}'`

Comment (ttodua) [reply ts/src/bingx.ts:1910]: please re-test now, i had to use `replaceAll` instead of `replace`.

**Pull request merged**